src/app/store/index.ts

```diff
@@ -37,14 +38,15 @@ const rootReducer = combineReducers({
   networks: networksSlice.reducer,
   onboarding: onboardingSlice.reducer,
   submittedTransactions: submittedTransactionsSlice.reducer,
+  theme: userSelectedThemeSlice.reducer,
```

Comment:
I think it might be more scalable if we call this slice `settings` or something similar to allow more state to be managed by it in the future?

Reply:
I'm not sure that adding a `settings` reducer without further changes would help. Some of the other reducers seem like settings too, like `networks`, and maybe `analytics`. If a settings reducer is created, without refactoring the other existing reducers, wouldn't it be leaving out some of the other "settings"?

 Perhaps we could refactor the store to better account for "settings", although given it's current size, having an extra `theme` reducer seems acceptable.

Reply:
`networks` uses `createEntityAdaptor` to manage adding/removing networks, so it was done as its own slice. `analytics` is not tracking a user setting. With persisted state you have to run migrations if you ever want to refactor those pieces of state ...I learned that with `onboarding` which was terribly implemented. Haha.

Reply:
Alright, will move it into `settings`

Reply:
There is a lingering old atom in `onboarding` that could be relocated and handled by this bc it is also a user setting that needs to be persisted, `hasAllowedDiagnosticsState`. Would be great to remove that `atomWithStorage`.